lib/validic/response.rb

```diff
@@ -11,5 +13,23 @@ def initialize(summary, response)
         @attributes = response
       end
     end
+
+    def next
+      extend Validic::REST::Utils
```

Comment:
never seen `extend` on a method before. normally you put `extend` at the class level